### PR TITLE
demo page with ACE editor

### PR DESCRIPTION
**Pull request by SashaOrlova**
#4 
Simple html with table and ACE editor. For open it use https://github.com/ajaxorg/ace-builds/

### CODE REVIEW

File: demo/example.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <td id="leftcol">
+      <pre id="editor">
+    \documentclass[12pt]{article}
+
+    \usepackage{answers}
+    \usepackage{setspace}
+    \usepackage{graphicx}
+}</pre>
+
+<script src="src-noconflict/ace.js" charset="utf-8"></script>
```
Comment (dbarashev) [inline demo/example.html]: No such file in the repository.

File: demo/example.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<script>
```
Comment (dbarashev) [inline demo/example.html]: Normally you don't want to mix `script` tags with other HTML markup. Move it to the end of document, right before the closing `</html>`

**Comment by SashaOrlova** [comment]
fixed

File: demo/example.html [hunk 3]
```diff
@@ -37,16 +37,16 @@
...
     </td>
     <td>right col</td>
    </tr>
   </table>
 
 </body>
-</html>
\ No newline at end of file
+</html>
+<script>
```
Comment (dbarashev) [inline demo/example.html]: Alexandra, did you read my comment properly?
>  right **before** the closing `</html>`

**Comment by SashaOrlova** [comment]
fix

File: demo/example.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    <td id="leftcol">
+      <pre id="editor">
+    \documentclass[12pt]{article}
+
+    \usepackage{answers}
+    \usepackage{setspace}
+    \usepackage{graphicx}
+}</pre>
+
+<script src="src-noconflict/ace.js" charset="utf-8"></script>
```
Comment (dbarashev) [inline demo/example.html]: Please move `<script>` tag from the middle of the document to the place where it is expected to be: to the end or to the `<head>` section.

Comment (dbarashev) [reply demo/example.html]: See [this perfect summary](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup) of script tag placement options.

**Review by dbarashev — APPROVED** [review]
LGTM. You can merge it now.

**Pull request merged**